website/docs/r/cs_kubernetes.html.markdown

```diff
@@ -53,13 +56,18 @@ The following arguments are supported:
 * `name` - The kubernetes cluster's name. It is the only in one Alicloud account.
 * `name_prefix` - The kubernetes cluster name's prefix. It is conflict with `name`. If it is specified, terraform will using it to build the only cluster name. Default to "Terraform-Creation".
 * `availability_zone` - (Force new resource) The Zone where new kubernetes cluster will be located. If it is not be specified, the value will be vswitch's zone.
-* `vswitch_id` - (Force new resource) The vswitch where new kubernetes cluster will be located. If it is not specified, a new VPC and VSwicth will be built. It must be in the zone which `availability_zone` specified.
+* `vswitch_id` - (Deprecated from version 1.16.0)(Force new resource) The vswitch where new kubernetes cluster will be located. If it is not specified, a new VPC and VSwicth will be built. It must be in the zone which `availability_zone` specified.
+* `vswitch_ids` - (Force new resource) The vswitch where new kubernetes cluster will be located. For SingleAZ Cluster, if it is not specified, a new VPC and VSwicth will be built. It must be in the zone which `availability_zone` specified. For MultiAZ Cluster, you must create three vswitches firstly, specify them here.
 * `new_nat_gateway` - (Force new resource) Whether to create a new nat gateway while creating kubernetes cluster. Default to true.
-* `master_instance_type` - (Required, Force new resource) The instance type of master node.
-* `worker_instance_type` - (Required, Force new resource) The instance type of worker node.
+* `master_instance_type` - (Deprecated from version 1.16.0)(Required, Force new resource) The instance type of master node.
+* `master_instance_types` - (Required, Force new resource) The instance type of master node. Specify one type for single AZ Cluster, three types for MultiAZ Cluster.
+* `worker_instance_type` - (Deprecated from version 1.16.0)(Required, Force new resource) The instance type of worker node.
+* `worker_instance_types` - (Required, Force new resource) The instance type of worker node. Specify one type for single AZ Cluster, three types for MultiAZ Cluster.
 * `worker_number` - The worker node number of the kubernetes cluster. Default to 3. It is limited up to 50 and if you want to enlarge it, please apply white list or contact with us.
 * `password` - (Required, Force new resource) The password of ssh login cluster node.
-* `pod_cidr` - (Required, Force new resource) The CIDR block for the pod network. It will be allocated automatically when `vswitch_id` is not specified.
+* `keypair` - (Required, Force new resource) The keypair of ssh login cluster node, you have to create it first.
```

Comment:
The field 'keypair' is not defined as "forcenew". Please keep consistent. Seem as other field.